core/src/main/java/gov/nist/secauto/metaschema/core/model/xml/XmlMetaConstraintLoader.java

```diff
@@ -61,10 +62,11 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.Nullable;
 
-public class MetaConstraintLoader
-    extends AbstractLoader<IConstraintSet> {
+public class XmlMetaConstraintLoader
+    extends AbstractLoader<IConstraintSet>
+    implements IConstraintLoader {
   // private static final Logger LOGGER =
-  // LogManager.getLogger(MetaConstraintLoader.class);
+  // LogManager.getLogger(XmlMetaConstraintLoader.class);
```

Comment:
Non-blocking: is there a reason we don't want to implement the logger pattern on this class?